### DISCUSSION TITLE
Allow passing validators without arguments as strings instead of strings wrapped in an array

### DISCUSSION THREAD

**Discussion by DamianGlowala** [formkit/formkit#853] [Ideas]
Instead of:
```
:validation="[['required']]"
```
... why not simplify it to the following form with a simple check whether given validation item is a string or an array? I believe this can be achieved in a backwards-compatible way.
```
:validation="['required']"
```

**Comment by fenilli** [comment]
This would work yes, but lose a bit of consistency, if there are more `['required', ['min', 20]]` it starts to get a bit strange as "why required isn't needed, but min is required to be []?".

Might be worth looking into it to make use of the current syntax, so `['required', 'min:20']` would also work, so cases where you need more you could use the array syntax, still think it looks ugly tho haha. @justin-schroeder what do you think?

**Comment by DamianGlowala** [comment]
Totally get your point about wondering why one is a string and the other is an array, even though to me it feels natural: we would only need to wrap a validator into an array to pass any arguments it needs. `min:20` could work, but imagine passing a variable instead of `20`... (I believe this is what this alternative syntax was created for.) 😄

**Reply by fenilli** [reply]
Its mostly because some regex are impossible with string literals haha but yes passing a variable is also a good thing ( you can do that with string templates too tho `min:${value}` ).
